README.md

```diff
@@ -1,59 +1,111 @@
 # drupal-check [![Build Status](https://travis-ci.com/mglaman/drupal-check.svg?branch=master)](https://travis-ci.com/mglaman/drupal-check) [![Latest release](https://img.shields.io/github/release/mglaman/drupal-check.svg)](https://github.com/mglaman/drupal-check/releases/latest)
 
+Built on [phpstan](https://github.com/phpstan/phpstan), this static analysis tool will check for correctness (e.g. using a class that doesn't exist), deprecation errors, and more.
+
+Why? While there are many static analysis tools out there, none of them run with the Drupal context in mind. This allows checking contrib modules for deprecation errors thrown by core.
+
+Are you ready for Drupal 9? Check out our [Drupal 9 Readiness](https://github.com/mglaman/drupal-check/wiki/Drupal-9-Readiness) instructions for details on how this tool can help.
+
 ## Requirements
 
-You must have PHP 7.1 or greater, which is a requirement of PHPStan, the static analyzer backing this tool.
+* PHP >=7.1
 
-## Usage
+## Installation
+
+The easiest way to install is by downloading the latest PHAR and putting it into your path. For example:
+
+```
+curl -O -L https://github.com/mglaman/drupal-check/releases/download/1.0.5/drupal-check.phar
+mv drupal-check.phar /usr/local/bin/drupal-check
+```
 
-For the best experience with this tool is best used against a vanilla Drupal project. It will work against custom Drupal 
-projects, but there can be conflicts against other development tools added to projects.
+### Composer
 
-Check deprecations
+You can also install this globally using composer like so:
 
 ```
-# Example: Against address contrib
-drupal-check /path/to/drupal8/modules/contrib/address
-drupal-check -d /path/to/drupal8/modules/contrib/address
+composer global require mglaman/drupal-check
 ```
 
-Check static analysis
+Refer to Composer's documentation on how to ensure global binaries are in your PATH: https://getcomposer.org/doc/00-intro.md#manual-installation.
+
+Note: you can also install this locally to your project and run it from that project's composer bin directory.
+
+### Build From Source
+
+You can also clone this repository and build the PHAR from source:
+
+First, install humbug/box:
 
 ```
-# Example: Against address contrib
-drupal-check -a /path/to/drupal8/modules/contrib/address
+composer global require humbug/box
 ```
 
-Check static analysis and deprecations
+Next, clone the repo and build:
 
 ```
-# Example: Against address contrib
-drupal-check -ad /path/to/drupal8/modules/contrib/address
+git clone https://github.com/mglaman/drupal-check.git
+cd drupal-check
+composer install --prefer-dist
+box -v
 ```
 
-Coming soon: code style w/ phpcs integration.
+## Usage
 
-## Install
+This tool works on contrib modules within a Drupal repository.
```

Comment:
It works for _all_ Drupal code. Currently it must execute within the root directory of a Drupal project